drivers/bridge/setup_ip_tables.go

```diff
@@ -171,3 +173,51 @@ func setIcc(bridgeIface string, iccEnable, insert bool) error {
 
 	return nil
 }
+
+// Control Inter Network Communication
+func setINC(network1, network2 string, enable bool) error {
+	var (
+		table  = iptables.Filter
+		chain  = "FORWARD"
+		args   = [2][]string{{"-s", network1, "-d", network2, "-j", "DROP"}, {"-s", network2, "-d", network1, "-j", "DROP"}}
+		buffer bytes.Buffer
+	)
+
+	if enable {
+		for i := 0; i < 2; i++ {
+			if iptables.Exists(table, chain, args[i]...) {
+				continue
+			}
+			if output, err := iptables.Raw(append([]string{"-I", chain}, args[i]...)...); err != nil {
+				buffer.WriteString("unable to add inter-network communication: ")
```

Comment:
Mmmh, there was a reason, which I can't recollect at the moment.
Maybe I did not want to abort if isolation is broken for one netwtork to one network only ?
If I can't remember a valid reason, I will just return error right away, as you suggested.